### PR TITLE
backport pyopenms-wheels to 2.8

### PR DESCRIPTION
**Pull request by poshul**
## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: .github/workflows/openms-ci.yml [hunk 1]
```diff
@@ -224,7 +222,7 @@ jobs:
           mkdir $GITHUB_WORKSPACE/OpenMS/bld/
           ## TODO allow changing of ctest build site in cmake
           ## TODO potentially parse some OS infos and add to build name
-          ctest --output-on-failure -V -S $GITHUB_WORKSPACE/OpenMS/tools/travis/cibuild.cmake
+          ctest --output-on-failure -V -S $GITHUB_WORKSPACE/OpenMS/tools/ci/cibuild.cmake
```
Comment (jpfeuffer) [inline .github/workflows/openms-ci.yml:225]: this will not work. on 2.8 it is still called travis

File: .github/workflows/openms-ci.yml [hunk 2]
```diff
@@ -332,7 +328,7 @@ jobs:
             ## TODO potentially parse some OS infos and add to build name
             ## TODO allow changing Contrib path to get rid of the extra cmake file!
             dbus-uuidgen > /var/lib/dbus/machine-id
-            xvfb-run ctest --output-on-failure -V -S $GITHUB_WORKSPACE/OpenMS/tools/travis/cibuild.cmake
+            xvfb-run ctest --output-on-failure -V -S $GITHUB_WORKSPACE/OpenMS/tools/ci/cibuild.cmake
```
Comment (jpfeuffer) [inline .github/workflows/openms-ci.yml:331]: travis

File: .github/workflows/pyopenms-wheels.yml [hunk 3]
```diff
@@ -105,7 +105,7 @@ jobs:
             pip install -U setuptools
             pip install -U pip
             pip install -U autowrap
-            pip install -U nose
+            pip install -U pytest
```
Comment (jpfeuffer) [inline .github/workflows/pyopenms-wheels.yml:108]: back then we still had nose

File: .github/workflows/pyopenms-wheels.yml [hunk 4]
```diff
@@ -245,7 +245,7 @@ jobs:
...
```
Comment (jpfeuffer) [inline .github/workflows/pyopenms-wheels.yml:248]: nose

**Comment by poshul** [comment]
At this point I think we are better ignoring this and focusing on 3.0

**Pull request closed without merging**